cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -237,3 +237,7 @@ spec:
         disabled: false
       osd:
         disabled: false
+
+  security:
```

Comment:
The kms only works for cluster-on-pvc, right?

Reply:
Yes, let's now expose it anywhere else yet.